sdk/tests/conformance/offscreencanvas/context-attribute-preserve-drawing-buffer.html

```diff
@@ -53,19 +61,45 @@
   gl.clearColor(1, 0, 1, 1);
   gl.clear(gl.COLOR_BUFFER_BIT);
 
-  await nextFrame();
-  var pixels = new Uint8Array(4);
-  wtu.checkCanvas(gl, color, msg);
+  var t, t0 = await nextFrame();
+  for (var i = 0, len = 5; i < len; i++) {
```

Comment:
This only performs at most 5 iterations. I thought the idea was to keep iterating until the first failure, or the timeout occurred (as the code intends to below). Was this leftover from an earlier version of the code?

Reply:
Oh I thought I misunderstood this at first. Will fix.